services/ui-src/src/views/NotFound/NotFound.tsx

```diff
@@ -52,15 +59,24 @@ const sx = {
   },
   warningIcon: {
     boxSize: "2rem",
+    "&.mobile": {
+      boxSize: "1.5rem",
+    },
   },
   headerText: {
     fontSize: "4xl",
     fontWeight: "normal",
+    "&.mobile": {
+      fontSize: "1.75rem",
```

Comment:
If this isn't one of our standard sizes, we can maybe push back on design about it, since it's possible they didn't realize they were going outside the normal font sizes.

Reply:
👍